src/uct/ib/dc/dc_mlx5.c

```diff
@@ -73,6 +73,17 @@ ucs_config_field_t uct_dc_mlx5_iface_config_sub_table[] = {
      ucs_offsetof(uct_dc_mlx5_iface_config_t, tx_policy),
      UCS_CONFIG_TYPE_ENUM(uct_dc_tx_policy_names)},
 
+    {"LAG_PORT_AFFINITY", "auto",
+     "Specifies how DCI select port under RoCE LAG. The values are:\n"
+     " auto     If the auto detected lag mode is queue_affinity,\n"
+     "          set DCI port affinity explicitly.\n"
+     "          For the other auto detected lag mode, e.g. hash,\n"
+     "          do not set DCI port affinity explicitly.\n"
```

Comment:
auto   Set DCI QP port affinity only if the hardware is configured to QUEUE_AFFINITY mode.
on      Always set DCI QP port affinity.
off      Never set DCI QP port affinity.

Reply:
Done